### PR TITLE
Add GET /models endpoint.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: daybed/tests/test_backend.py [hunk 1]
```diff
@@ -52,6 +52,31 @@ def test_add_token_fails_if_already_exist(self):
...
+        self.assertEqual(set(remy_models), set(["rems", "modelname"]))
+
+        alex_models = self.db.get_models(["Alexis"])
+        self.assertEqual(len(alex_models), 2)
+        self.assertEqual(set(alex_models), set(["alex", "modelname"]))
+
+        principal_models = self.db.get_models(["Remy", "Alexis"])
+        self.assertEqual(len(principal_models), 3)
+        self.assertEqual(set(principal_models), set(["rems", "alex", "modelname"]))
+
```
Comment (leplatrem) [inline daybed/tests/test_backend.py:98]: Could you please a few comments around to explain you assertions ?

**Comment by Natim** [comment]
@leplatrem updated.

File: daybed/backends/redis/__init__.py [hunk 2]
```diff
@@ -150,7 +165,11 @@ def delete_model(self, model_id):
         doc = self.__get_raw_model(model_id)
         doc["records"] = self.delete_records(model_id)
         self._db.delete("model.%s" % model_id)
-        return doc
+        return {
+            "definition": doc["definition"],
+            "records": doc["records"],
+            "acls": doc["acls"]
```
Comment (leplatrem) [inline daybed/backends/redis/__init__.py]: Caution when rebasing `acls` is now permissions

File: daybed/tests/test_backend.py [hunk 3]
```diff
@@ -52,6 +52,53 @@ def test_add_token_fails_if_already_exist(self):
...
+        # where at least one of them can read_definition de-duplicated
+        principal_models = self.db.get_models(["Remy", "Alexis"])
+        self.assertEqual(len(principal_models), 3)
+        self.assertEqual(sorted(principal_models, key=lambda x: x["id"]), [
+            {"id": "alex", "title": "Simple Alex",
+             "description": "One optional field"},
+            {"id": "modelname", "title": "simple",
+             "description": "One optional field"},
+            {"id": "rems", "title": "Simple Remy",
+             "description": "One optional field"}])
```
Comment (leplatrem) [inline daybed/tests/test_backend.py:97]: It's usually recommanded to write small test methods, inside a dedicated test suite, taking advantage of `setUp` mechanisms.

This way, the specifications of what is being implemented is much clearer (`def test_models_is_present_when_read_permission...`), and it's much easier to find out what behaviour was broken when one of the small tests fails...

**Pull request merged**